### PR TITLE
Update group_three.md

### PR DESCRIPTION
**Pull request by ciarabujanos**

### CODE REVIEW

**Comment by ciarabujanos** [comment]
@emhickmann

File: m4_sessions/1708-inning/group_three.md [hunk 1]
```diff
@@ -3,18 +3,29 @@
...
-* takeaways/learnings from those experiences
-* why is important to think about failure - normalize and be comfortable with it
+* intros (5 mins)
+* choose 1 article to read (10 min)
+https://www.scientificamerican.com/article/getting-it-wrong/
+https://www.psychologytoday.com/blog/the-squeaky-wheel/201501/10-surprising-facts-about-failure
+* what does failure mean to you? (everyone) (5 mins)  
+* break into 4 groups, what are some feelings/stigmas associated with failure?  (small groups, then everyone) (10 mins)
+* during this time we could talk about our failures too. then in the small group have them break into pairs and think about a time they applied themselves to something and came up short? how did you deal with ins) (6-8 mins)
+* Then everybody ask how have they dealt with failure in the past? Good practices/bad practices (everyone 5 mins)
```
Comment (emhickmann) [inline m4_sessions/1708-inning/group_three.md:25]: Be sure to put this on the board or share so people can learn from each other

File: m4_sessions/1708-inning/group_three.md [hunk 2]
```diff
@@ -3,18 +3,29 @@
...
 
+### Description: 
+* It is impossible to live without failing at something, unless you live so cautiously that you might as well not have lived at all, in which case you have failed by default.” — J.K. Rowling
+Why Failure is so Important
+
 ### Session Outcomes: 
-* Convince people that failure can be a good thing, especially for personal growth when you deal with it in the right way. 
+Students will leave this session:
+
+1)Being able to identify the benefits of failing
```
Comment (emhickmann) [inline m4_sessions/1708-inning/group_three.md:13]: Very clear outcomes!

File: m4_sessions/1708-inning/group_three.md [hunk 3]
```diff
@@ -3,18 +3,29 @@
...
-* share some of our failures
-* think about a time they failed or took a great risk and thought about failing
-* how have they dealt with failure in the past
-* takeaways/learnings from those experiences
-* why is important to think about failure - normalize and be comfortable with it
+* intros (5 mins)
+* choose 1 article to read (10 min)
+https://www.scientificamerican.com/article/getting-it-wrong/
+https://www.psychologytoday.com/blog/the-squeaky-wheel/201501/10-surprising-facts-about-failure
+* what does failure mean to you? (everyone) (5 mins)
```
Comment (emhickmann) [inline m4_sessions/1708-inning/group_three.md:22]: Are you having them write or pair share?

File: m4_sessions/1708-inning/group_three.md [hunk 4]
```diff
@@ -3,18 +3,29 @@
...
+* intros (5 mins)
+* choose 1 article to read (10 min)
+https://www.scientificamerican.com/article/getting-it-wrong/
+https://www.psychologytoday.com/blog/the-squeaky-wheel/201501/10-surprising-facts-about-failure
+* what does failure mean to you? (everyone) (5 mins)  
+* break into 4 groups, what are some feelings/stigmas associated with failure?  (small groups, then everyone) (10 mins)
+* during this time we could talk about our failures too. then in the small group have them break into pairs and think about a time they applied themselves to something and came up short? how did you deal with ins) (6-8 mins)
+* Then everybody ask how have they dealt with failure in the past? Good practices/bad practices (everyone 5 mins)
+* takeaways/learnings from those experiences. what are the benefits of failing? (10 mins)
+potential to show 12 famous people who failed then succeeded: (https://www.wanderlustworker.com/12-famous-people-who-failed-before-succeeding/)
```
Comment (emhickmann) [inline m4_sessions/1708-inning/group_three.md:27]: I think you should absolutely make time for this video.  I think it brings it all back together.  And maybe even start with the stat they show at the end of the video.

**Review by emhickmann — COMMENTED** [review]
I'm sure you know who is doing what part of the sessions so you don't have to add that in.  I might have questions on the board to guide discussion  @buji405

**Comment by emhickmann** [comment]
@buji405 Added a few comments for your group!

**Pull request merged**